source/index.html.md

```diff
@@ -538,6 +538,101 @@ ID | The ID of the talent agency to retrieve
 
 See example response to the right for an overview of included fields.
 
+# Blog posts
+
+## Get all blog posts
+
+```shell
+curl "https://www.filmmakers.eu/api/v1/blog_posts" \
+  -H "Authorization: Token token=API_KEY"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "id": 36,
+    "title": "Blog post title"
+  },
+  {
+    "id": 35,
+    "title": "Blog post title"
+  }
+]
+
+```
+
+This endpoint retrieves all blog posts available with the access rights of the API key.
+
+### Pagination
+
+This API endpoint is paginated, following the proposed [RFC-8288](https://tools.ietf.org/html/rfc8288) standard for Web linking.
+
+The pagination information is included in response HTTP headers, namely `Total` (total entries), `Per-Page` (entries per page) and `Link` (containing links to corresponding pages). The current page can be set using the `page`-parameter.
+
+### HTTP Request
+
+`GET https://www.filmmakers.eu/api/v1/blog_posts`
+
+### Query Parameters
+
+Parameter | Default | Description
+--------- | ------- | -----------
+page | 1 | Page to display - see "Pagination" above
+
+### Response fields
+
+Field | Type | Description
+--------- | ------- | -----------
+id | number | Unique ID of the blog post
+title | string | Title of the blog post
+
+## Get a specific blog post
+
+```shell
+curl "https://www.filmmakers.eu/api/v1/blog_posts/123" \
+  -H "Authorization: Token token=API_KEY"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "id": 35,
+  "title": "Blog post title",
+  "body": "Blog post body",
+  "body_html": "<div>    Blog post body    <action-text-attachment content-type=\"image/jpeg\" url=\"https://www.filmmakers.eu/rails/active_storage/blobs/redirect/abc/image.jpg\" filename=\"image.jpg\" filesize=\"407594\" width=\"2048\" height=\"1910\" previewable=\"true\" presentation=\"caption\" caption=\"Acme inc\">      <figure class=\"attachment attachment--preview attachment--jpg\">        <div class=\"image-wrapper\">          <img src=\"https://www.filmmakers.eu/rails/active_storage/representations/redirect/abc/image.jpg\">          <div class=\"image-caption\"><span title=\"© Acme inc\">© Acme inc</span></div>        </div>      </figure>    </action-text-attachment>  </div>",
+  "images": [
+    {
+      "url": "https://www.filmmakers.eu/rails/active_storage/representations/redirect/abc/image.jpg",
+      "copyright": "Acme inc"
+    }
+  ]
+}
+
+
+
+```
+
+This endpoint retrieves a specific blog post.
+
+### HTTP Request
+
+`GET https://www.filmmakers.eu/api/v1/blog_posts/<ID>`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+ID | The ID of the blog post to retrieve
+
+### Response fields
+
+See example response to the right for an overview of included fields. Please note:
```

Comment:
I think we need to list them explicitly as there are conditionals that might not be self explanatory (e.g. mainly around the source). So let's add a table here as well please.